cmd/sysl/cmd_ints.go

```diff
@@ -1,60 +1,14 @@
 package main
 
 import (
-	"regexp"
+	"github.com/anz-bank/sysl/pkg/integrationdiagram"
```

Comment:
Thoughts on renaming the package to `intdiagram` or `intdiag` rather than `integrationdiagram` ?

Reply:
Nope, I like integrationdiagram
does ints mean integer? does it mean integration?
indegrationdiagram is nice and unique

same goes for sd; sequencediagram is very descriptive, which why that package name isn't sd